### PR TITLE
KEP-4633: Graduate to BETA.

### PR DESCRIPTION
**Pull request by vinayakankugoyal**
### Description

Update the stage from Alpha to Beta for featuregate AnonymousAuthConfigurableEndpoints in kubernetes documentation.

KEP: https://github.com/kubernetes/enhancements/issues/4633
PR(s): https://github.com/kubernetes/kubernetes/pull/127009

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates/anonymous-auth-configurable-endpoints.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ stages:
   - stage: alpha 
     defaultValue: false
     fromVersion: "1.31"
+    toVersion: "1.31"
+  - stage: beta 
+    defaultValue: true
+    fromVersion: "1.32"
 ---
 Enable [configurable endpoints for anonymous auth](/docs/reference/access-authn-authz/authentication/#anonymous-authenticator-onfiguration)
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/anonymous-auth-configurable-endpoints.md:17]: For beta, we ask for a higher quality bar. Please also fix the link - https://kubernetes.io/docs/reference/access-authn-authz/authentication/#anonymous-authenticator-onfiguration doesn't work - and check that the page this links to would be up to date.

Comment (vinayakankugoyal) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/anonymous-auth-configurable-endpoints.md]: > For beta, we ask for a higher quality bar.

Can you elaborate on what you are asking? Is this comment meant to ask me to update the incorrect link?

Comment (sftim) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/anonymous-auth-configurable-endpoints.md]: What we (SIG Docs) would like, as part of the graduation to beta, is that you / SIG Auth check through the existing documentation and make sure it's beta quality or better.

This is typically:
- fix any broken links etc
- align with the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) (broadly - doesn't mean get every detail perfect)
- ensure all the associated docs are factually correct

Generally, the docs should be written as if the feature is GA, but then add caveats about enabling behavior if needed.

These could be more than one PR. For this PR, yes: please fix the broken link.

Comment (vinayakankugoyal) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/anonymous-auth-configurable-endpoints.md]: Thanks, fixed the broken links and took a pass at https://kubernetes.io/docs/reference/access-authn-authz/authentication/#anonymous-authenticator-configuration, it LGTM, but happy to update it if it doesn't meet the style guide. That is the only doc for this feature, so we should be good to go?

**Review by sftim — COMMENTED** [review]
@vinayakankugoyal, you will also need to update https://kubernetes.io/docs/reference/access-authn-authz/authentication/#anonymous-authenticator-configuration to reflect that the feature gate is enabled by default.

**Comment by vinayakankugoyal** [comment]
> @vinayakankugoyal, you will also need to update https://kubernetes.io/docs/reference/access-authn-authz/authentication/#anonymous-authenticator-configuration to reflect that the feature gate is enabled by default.

I thought that is what the banner for the feature-gate is for.

**Comment by vinayakankugoyal** [comment]
@sftim - could we update the shortcode here: https://github.com/kubernetes/website/blob/main/layouts/shortcodes/feature-state.html to display the default value instead? having to update the docs feels moot given that we have that shortcode?

**Comment by vinayakankugoyal** [comment]
@sftim - the shortcode changes we made are in dev-1.32 now. So the enabled by default info is already displayed. I also did a pass at the [existing](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#anonymous-authenticator-configuration) docs for this feature. Thanks for your review!

**Comment by sftim** [comment]
@vinayakankugoyal, you will also need to update https://kubernetes.io/docs/reference/access-authn-authz/authentication/#anonymous-authenticator-configuration to remove some of the existing text:

> To enable configuring anonymous auth via the config file you need enable the `AnonymousAuthConfigurableEndpoints` feature gate.

**Comment by vinayakankugoyal** [comment]
> @vinayakankugoyal, you will also need to update https://kubernetes.io/docs/reference/access-authn-authz/authentication/#anonymous-authenticator-configuration to remove some of the existing text:
> 
> > To enable configuring anonymous auth via the config file you need enable the `AnonymousAuthConfigurableEndpoints` feature gate.

Oh good call. Done! Thanks!

**Pull request merged**